components/SizerBase/src/SizerBase.xaml

```diff
@@ -49,33 +66,58 @@
             <Setter.Value>
                 <ControlTemplate TargetType="controls:SizerBase">
                     <Grid x:Name="RootGrid"
-                          Background="{TemplateBinding Background}">
-
+                          Background="{TemplateBinding Background}"
+                          BorderBrush="{TemplateBinding BorderBrush}"
+                          BorderThickness="{TemplateBinding BorderThickness}">
+                        <Grid.BackgroundTransition>
+                            <BrushTransition Duration="0:0:0.083" />
```

Comment:
Ooo, didn't know this was a thing, cool.

Reply:
Yeah, makes all the state transitions in WinUI 2.6+ so much smoother 😊